src/constants/responses.ts

```diff
@@ -55,3 +55,11 @@ export const TASKS_FETCH_FAILED = "An error occurred while fetching tasks.";
 export const FAILED_TO_FETCH_TASKS = `Failed to fetch tasks for **{{assignee}}**.`;
 export const USER_NOT_FOUND = `User Not Found`;
 export const USER_STATUS_NOT_FOUND = "No Status Found";
+
+export const OVERDUE_DEFAULT_MESSAGE = "You have overdue tasks.";
+export const OVERDUE_CUSTOM_MESSAGE =
+  "Please be aware that you currently have tasks that are overdue or due within the next {{days}} day. If you require additional time to complete these tasks, kindly submit an extension request.";
```

Comment:
nit: can we give custom message a little short and crisp?

Reply:
Example?

Reply:
**export const OVERDUE_CUSTOM_MESSAGE =** "**Alert !** Your task will be overdue within 2/1 days(if >1)/day(if 1). You may consider raising an extension request if needed."